packages/security/src/components/StatusIcon/StatusIcon-story.js

```diff
@@ -23,6 +24,27 @@ const storyProps = () => ({
 const status = STATUS[0];
 
 storiesOf(components('StatusIcon#legacy'), module)
+  .addDecorator((Story) => (
+    <>
+      <InlineNotification
+        className="page-layouts__banner"
+        actions={
+          <NotificationActionButton
+            href="https://carbon-for-ibm-products.netlify.app/?path=/docs/ibm-products-components-cards-productivecard--default"
```

Comment:
Can we double check this link? I think this one should probably follow Nav and just say it’s now legacy because I don’t see an updated link in the Box note @cameroncalder shared. In which case, we can remove the `NotificationActionButton` altogether.